packages/relay/src/routes/reportRoute.ts

```diff
@@ -0,0 +1,31 @@
+import { DB } from 'anondb/node'
+import { Express, Request, Response } from 'express'
+import { UnirepSocialSynchronizer } from '../services/singletons/UnirepSocialSynchronizer'
+import { errorHandler } from '../services/utils/ErrorHandler'
+import { reportService } from '../services/ReportService'
+
+export default (
+    app: Express,
+    db: DB,
+    synchronizer: UnirepSocialSynchronizer
+) => {
+    app.post(
+        '/api/report/create',
```

Comment:
should modify to '/api/report'

Reply:
Should we make it create so the path is clearer?

Reply:
my understanding for RESTful pattern is that path is for the resource (model, entity) `/api/resource` we want to interact with the database. Then we use request methods(POST, GET, PUT, DELETE) to tell server side what operation we are about to interact with the resource.

so, in my opinion, POST `api/report` is clear to express that I want to insert a new report into the database.